### DISCUSSION TITLE
Filter by best-per-language

### DISCUSSION THREAD

**Discussion by mmcdon20** [PlummersSoftwareLLC/PrimeView#22] [Ideas]
This is a feature request.

Currently there are over 30 runs for C++ alone. C, C#, Rust, and Zig, each have about 20 runs. It would be nice if we could filter the output by the best run per language to get a better understanding of where the languages rank relative to each other.

**Comment by rbergen** [comment]
> This is a feature request.

In that case I'm converting it to an idea discussion.

**Reply by rbergen** [reply]
You can shorten the list(s) significantly by selecting the Leaderboard filter preset. That filter corresponds with the one Dave applies to establish the "official" ranking in the context of the YouTube series.

**Reply by mmcdon20** [reply]
Yes, you can narrow it down with the leaderboard preset, or other combinations of filters, but even in the leaderboard preset C# is on there 8 times for example.

**Comment by rbergen** [comment]
After thinking about it for a bit, I concluded it would be a fun feature to figure out and code. I've opened #23 to add it.

**Reply by rbergen** [reply]
#23 was merged a couple of hours ago.
